src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -88,9 +94,14 @@
 
     private static final Log LOG = LogFactory.getLog(Worker.class);
 
+    // Default configs for periodic shard sync
     private static final int SHARD_SYNC_SLEEP_FOR_PERIODIC_SHARD_SYNC = 0;
     private static final int MAX_INITIALIZATION_ATTEMPTS = 20;
     private static final int PERIODIC_SHARD_SYNC_MAX_WORKERS_DEFAULT = 1; //Default for KCL.
+    static final long LEASE_TABLE_CHECK_FREQUENCY_MILLIS = 3 * 1000L;
```

Comment:
Should we expose these as configurable for the customer? I know it's not in 2.x but I'm wondering if this is something the customer might want to configure - say the customer's application can't tolerate a 30 second gap of delay for their application / or can tolerate a longer delay of processing.

Reply:
This is a good point, I was thinking about that. The concern is that we already expose an overwhelming amount of configs, and I'm not sure there's what the customer demand for tuning this would be. 

On the plus side, this would make the unit tests run a lot faster if this was in a config. We can sync up with Ashwin for a final decision, but for now will leave that change out of this PR to keep consistent with 2.x.

Reply:
Yeah I don't think we should block on this, this is in sync with 2.x, I think this should be user configurable so we can always add this as a follow-up task after this PR.